.travis.yml

```diff
@@ -20,12 +16,72 @@ env:
         - SYLIUS_CACHE_DIR=$HOME/.sylius-cache
         - SYLIUS_BUILD_DIR=etc/build
 
+matrix:
+    include:
+        # PHP 7.2 & Sylius 1.3
+        -
+            php: 7.2
+            env:
+                - SYLIUS_VERSION="1.3"
+                - SYMFONY_VERSION="3.4.*"
+        -
+            php: 7.2
+            env:
+                - SYLIUS_VERSION="1.3"
+                - SYMFONY_VERSION="4.1.*"
+        -
+            php: 7.2
+            env:
+                - SYLIUS_VERSION="1.3"
+                - SYMFONY_VERSION="4.2.*"
+        -
+            php: 7.2
+            env:
+                - SYLIUS_VERSION="1.3"
+                - SYMFONY_VERSION="4.3.*"
+                
+        # PHP 7.2 & Sylius 1.4
+        -
+            php: 7.2
+            env:
+                - SYLIUS_VERSION="1.4"
+                - SYMFONY_VERSION="3.4.*"
+        -
+            php: 7.2
+            env:
+                - SYLIUS_VERSION="1.4"
+                - SYMFONY_VERSION="4.1.*"
+        -
+            php: 7.2
+            env:
+                - SYLIUS_VERSION="1.4"
+                - SYMFONY_VERSION="4.2.*"
+        -
+            php: 7.2
+            env:
+                - SYLIUS_VERSION="1.4"
+                - SYMFONY_VERSION="4.3.*"
+
+        # PHP 7.3 & Sylius 1.4
+        -
+            php: 7.3
+            env:
+                - SYLIUS_VERSION="1.4"
+                - SYMFONY_VERSION="4.2.*"
+        -
+            php: 7.3
+            env:
+                - SYLIUS_VERSION="1.4"
+                - SYMFONY_VERSION="4.3.*"
+
```

Comment:
You are missing Sylius 1.5

Reply:
That was temporary set. Final set have 1.4, 1.5 and latest/default (without version specified) - I've played with `.travis.yml` a lot with `--force` pushes.